buildscripts/libs/build_json.sh

```diff
@@ -37,7 +37,7 @@ url="https://github.com/nlohmann/json/archive/$tarfile"
 [[ -d build ]] && rm -rf build
 mkdir -p build && cd build
 
-cmake .. -DCMAKE_INSTALL_PREFIX=$prefix -DJSON_BuildTests=$MAKE_CHECK
```

Comment:
On oeion, this installs to lib64 by default, which doesn't work with the hardcoded name "lib" in the schema validator build.